apps/admin/src/components/users/table/users-table-columns.tsx

```diff
@@ -0,0 +1,124 @@
+"use client";
+
+import {Button} from "@energyleaf/ui";
+import type {ColumnDef} from "@tanstack/react-table";
+import {
+    BanIcon,
+    CheckCircle2Icon,
+    ChevronDownIcon,
+    ChevronUpIcon
+} from "lucide-react";
+import UserActionCell from "@/components/users/table/user-action-cell";
+
+export interface UserTableType {
+    id: number;
+    username: string;
+    mail: string;
+    active: boolean;
```

Comment:
Ich hab´s jetzt schon öfters gesehen und hab mich gefragt: Macht es Sinn die Variable noch in isActive umzubennen? Dann wird auch im Code direkt klar, dass es eindeutig ein bool ist. Aber auch nur ne Kleinigkeit. Kannst auch so lassen

Reply:
Hast Recht, mit isActive wird das schneller klar. Passe ich an :+1: